README.md

```diff
@@ -1,5 +1,13 @@
-# kitops
-Open-source Python package for [KitOps'](https://kitops.ml) ModelKit management
+# PyKitOps
+
+PyKitOps is an open source Python SDK for managing [KitOps'](https://kitops.ml) ModelKits. KitOps is a packaging, versioning, and sharing system for AI/ML projects that uses open standards so it works with the AI/ML, development, and DevOps tools you are already using, and can be stored in your enterprise container registry. 
+
+PyKitOps makes it easy to create a KitOps' ModelKit for your AI/ML project directly in code. This makes PyKitOps preferred when assembling a ModelKit from:
```

Comment:
I'm not sure the apostrophe is grammatically right here -- we wouldn't say Docker's Container (cf. KitOps' ModelKit)

Reply:
I agree. I would write it as a "for managing KitOps ModelKits."